HelloWorld_ST_Sensors/user/gps/gps.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef USER_GPS_H_
+#define USER_GPS_H_
+
+#include "mbed.h"
+#include <string.h>
+
+namespace gps_ns
```

Comment:
Improve naming

namespace gps
class gps device name. For example 
- gps::ublox_[part_number]
- gps::adafruit_[part_number]